zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -836,7 +822,23 @@ public boolean run(String paragraphId) {
    * @param paragraphId ID of paragraph
    */
   public boolean run(String paragraphId, boolean blocking) {
+    return run(paragraphId, blocking, null);
+  }
+
+  /**
+   * Run a single para
+   * @param paragraphId
+   * @param blocking
+   * @param ctxUser
+   * @return
+   */
+  //fix for ZEPPELIN-3065 -- shoaib.rehman
```

Comment:
`//fix for ZEPPELIN-3065 -- shoaib.rehman`
I think this comment can be deleted because it doesn't make sense.

Reply:
Please delete  `//fix for ZEPPELIN-3065 -- shoaib.rehman`